### PR TITLE
1-to-1s blog

### PR DESCRIPTION
**Pull request by jarohen**
Hey folks - I've written a blog on 1-to-1 practices I've found useful - feedback welcome :slightly_smiling_face:

TODO: 
* [ ] Better title/subtitle
* [x] Hero image
* [ ] Discussion link - will add in a link to LI when it's published.

### CODE REVIEW

File: src/pages/blog/technical-121s.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  - culture
+  - development teams
+  - mentoring
+heroImage: 'technical-121s.jpg'
+---
+
+I've just come out of a JUXT team-leads catch-up where we've discussed the art of 1-to-1s, mostly to share experiences and best practices from our wide variety of clients and projects, but with the aim of eventually writing up an internal beginners' guide for new team-leads.
+
+Anyway, it got me thinking, reflecting on what I do and why.
+I highly doubt anything in here is particularly novel or ground-breaking, but I hope they're as useful to you as they have been to me!
```
Comment (refset) [inline src/pages/blog/technical-121s.md:19]: `they're` -> `these reflections` perhaps?

Comment (jarohen) [reply src/pages/blog/technical-121s.md:19]: @refset thanks, fair call to clarify - intended "they" as in "the ideas", if that makes any difference?

Comment (joelittlejohn) [reply src/pages/blog/technical-121s.md:19]: But "the ideas" aren't mentioned at the moment @jarohen. It's a floating pronoun.

Comment (jarohen) [reply src/pages/blog/technical-121s.md:19]: @joelittlejohn fair, yep :slightly_smiling_face: will be more explicit

**Pull request merged**